src/components/modals/StaticModalComponent.js

```diff
@@ -16,13 +16,17 @@ type Props = {
   cancel(): void
 }
 class StaticModalComponent extends Component<Props> {
+  reset: number
   componentDidMount () {
     if (this.props.modalDismissTimerSeconds) {
-      setTimeout(() => {
+      this.reset = setTimeout(() => {
```

Comment:
Have you confirmed what the return type `setTimeout` is?
I think it's an object, but `this.reset` is declared `number`

Reply:
why is this in `componentDidMount` and not `constructor`?